### ISSUE TITLE
Cannot update recipient's bank account using token

### ISSUE THREAD

**Issue by layby42** [stripe/stripe-go#184]
Based on https://stripe.com/docs/api#update_recipient it supposed to be possible to update recipient's bank account using token. It is possible via ruby version of stripe API.

In package recipient **New** method has

```
	if params.Bank != nil {
		if len(params.Bank.Token) > 0 {
			body.Add("bank_account", params.Bank.Token)
		} else {
			params.Bank.AppendDetails(body)
		}
	}
```

But **Update** method has only

```
		if params.Bank != nil {
			params.Bank.AppendDetails(body)
		}
```

Please fix **Update** method to accept Token

**Comment by brandur** [comment]
@layby42 Hi there! We're sorry to hear about the trouble here, and thanks for reporting the problem! I've added a patch for the issue in #185 and we should be able to get it into mainline soon.

**Comment by brandur** [comment]
Released as 11.1.1.
